next-tavla/src/Board/scenarios/Table/components/Via/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useNonNullContext } from 'hooks/useNonNullContext'
+import { DepartureContext } from '../../contexts'
+import classes from './styles.module.css'
+
+function Via() {
+    const departure = useNonNullContext(DepartureContext)
+
+    const viaDestinations = departure.destinationDisplay?.via
+
+    function stringifyViaDestinations(viaDest: typeof viaDestinations) {
+        if (!viaDest || !viaDest.length) return null
+
+        return viaDest.reduce((prev, next) => {
+            if (!prev) return next
+            if (!next) return prev
+            return prev + ', ' + next
+        })
+    }
```

Comment:
Jeg er i utgangspunktet ikke noe fan av å lage funksjoner for kode som kun skal kjøre en gang. Hver gang denne komponenten re-rendrer lager man en ny funksjon. Man kunne gjort all funksjonaliteten direkte istedenfor